lib/tasks/index.js

```diff
@@ -0,0 +1,7 @@
+var post = require('./post');
```

Comment:
Remove these `var` declarations. Now redundant and unused given the changes below.